test cases/common/82 add language/meson.build

```diff
@@ -2,9 +2,12 @@ project('add language', 'c')
 
 test('C', executable('cprog', 'prog.c'))
 
-assert(add_languages('cpp', native: false), 'Add_languages returned false on success')
+assert(add_languages('cpp', native: 'both'), 'Add_languages returned false on success')
```

Comment:
Shouldn't that fail when not cross compiling?

Reply:
Hm, better to accept it indeed, just like we allow `native: true` and `native: false` to mean the same thing when not cross compiling. But maybe that should be said in doc explicitly.